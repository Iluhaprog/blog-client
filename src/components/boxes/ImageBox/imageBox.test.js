import React from 'react';
import { render } from '@testing-library/react';
import { ImageBox } from '.';

test('Shoud render box without image', () => {
    const { asFragment } = render(<ImageBox>Test</ImageBox>);
    expect(asFragment()).toMatchSnapshot();
});


test('Shoud render box with image', () => {
    const { asFragment } = render(<ImageBox img='./img.png'>Test</ImageBox>);
    expect(asFragment()).toMatchSnapshot();
});