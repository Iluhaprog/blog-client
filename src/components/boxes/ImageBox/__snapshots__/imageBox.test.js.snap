// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Shoud render box with image 1`] = `
<DocumentFragment>
  <div
    class="image-box"
  >
    <img
      alt="image"
      src="./img.png"
    />
    <div
      class="image-box__inner"
    >
      <div
        class="column column_jc-c column_ai-c "
      >
        Test
      </div>
    </div>
  </div>
</DocumentFragment>
`;

exports[`Shoud render box without image 1`] = `
<DocumentFragment>
  <div
    class="image-box"
  >
    <div
      class="image-box__inner"
    >
      <div
        class="column column_jc-c column_ai-c "
      >
        Test
      </div>
    </div>
  </div>
</DocumentFragment>
`;
